drivers/include/at86rf215.h

```diff
@@ -71,11 +71,11 @@ typedef void (*at86rf215_batmon_cb_t)(void *arg);
  *          as a clock source on the board.
  *          Otherwise it is turned off to save energy.
  */
-#ifndef AT86RF215_USE_CLOCK_OUTPUT
-#define AT86RF215_USE_CLOCK_OUTPUT      (0)
+#ifndef CONFIG_AT86RF215_USE_CLOCK_OUTPUT
+#define CONFIG_AT86RF215_USE_CLOCK_OUTPUT   0
```

Comment:
This way it has the same behaviour as the Kconfig boolean:
```suggestion
#ifdef DOXYGEN
#define CONFIG_AT86RF215_USE_CLOCK_OUTPUT
```

Reply:
Thanks for the suggestion.

Reply:
Tested Successfully with above modifications.